ODLA/platforms/odla_popart/odla_compute.cc

```diff
@@ -51,7 +51,7 @@
 #endif
 
 thread_local odla_computation g_comp;
-static std::vector<std::unique_ptr<_odla_computation>> g_comps;
+thread_local std::vector<std::unique_ptr<_odla_computation>> g_comps;
```

Comment:
g_comps is supposed to be global. 
g_comp is TLS as it is meant for current computation.

Reply:
after test, it is not caused by g_comps